starknet_py/tests/e2e/conftest.py

```diff
@@ -457,3 +458,8 @@ async def cairo_serializer(gateway_account_client: AccountClient) -> CairoSerial
     contract = deployment_result.deployed_contract
 
     return CairoSerializer(identifier_manager=contract.data.identifier_manager)
+
+
+@pytest_asyncio.fixture(name="default_gateway_gas_price", scope="module")
+def default_gateway_gas_price():
+    return StarknetGeneralConfig().min_gas_price
```

Comment:
We are trying to describe all of our fixtures to exactly know what they are responsible for, so please do it here 🙏

Reply:
Also please add a return type :)